dipy/workflows/tests/test_io.py

```diff
@@ -91,3 +94,33 @@ def test_split_flow():
         split_data, split_affine = load_nifti(split_path)
         npt.assert_equal(split_data.shape, volume[..., 0].shape)
         npt.assert_array_almost_equal(split_affine, affine)
+
+
+def test_convert_sh_flow():
+    with TemporaryDirectory() as out_dir:
+        out_dir = Path(out_dir)
+        filepath_in = out_dir/'sh_coeff_img.nii'
```

Comment:
you should use `os.path.join(out_dir,'sh_coeff_img.nii')` 

I suppose, Windows CI are failing because of `/`

Reply:
ah thanks for catching this. I didn't know pathlib doesn't take the os differences into account

Reply:
I think there is a way to do it correctly with pathlib also but I need to look. maybe later today